src/test/java/org/arquillian/container/chameleon/ConfigurationTestCase.java

```diff
@@ -80,4 +80,22 @@ public void shouldFailOnMissingContainerFile() throws Exception {
         configuration.setChameleonTarget("wildfly:8.2.0.Final:managed");
         configuration.validate();
     }
+
+    @Test
+    public void shouldReturnFalseForMissingContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("MISSING_TARGET:8.2.0.Final:managed");
+        Assert.assertFalse(isSupported);
+    }
+
+    @Test
+    public void shouldReturnTrueForGivenContainerName() throws Exception {
+        boolean isSupported = ChameleonConfiguration.isSupported("wildfly:8.2.0.Final:managed");
```

Comment:
In theory and if I understood correctly everything from the point of view of user would be the same (`--container-adapter glassfish-embedded-3.1`) then if this container is chameleon aware Forge will translate it to Chameleon configuration, and if not then standard/old fashion is used.

This makes me think that maybe is supported method is not really required.